main.tf

```diff
@@ -645,6 +645,14 @@ module "container_definition_bitbucket" {
   mount_points             = var.mount_points
   volumes_from             = var.volumes_from
 
+  healthcheck = {
```

Comment:
I guess this should be also added to module `container_definition_github_gitlab` (line 565 - https://github.com/terraform-aws-modules/terraform-aws-atlantis/pull/265/files#diff-dc46acf24afd63ef8c556b77c126ccc6e578bc87e3aa09a931f33d9bf2532fbbL565)

Reply:
yep, missed that